src/__tests__/createLoggerMiddleware-test.js

```diff
@@ -167,11 +167,60 @@ describe('createLoggerMiddleware', () => {
       ],
       data: null
     });
+
     expect(stderrCalls[1].level).toBe('ERROR');
 
     logger.destroy();
   });
 
+  it('reduces the log payload of `variables` to the desired maxGraphQLVariablesLength', async () => {
+    const logger = new Logger({service: 'pizza-shop', isProduction: true});
+    const server = createServer(logger, {maxGraphQLVariablesLength: 12});
+
+    await supertest(server).get('/');
+    await supertest(server).get('/404');
+
+    await supertest(server)
+      .post('/graphql')
+      .send(mockGraphQLPayload);
+
+    const stdoutCalls = process.stdout.write.mock.calls.map(call =>
+      JSON.parse(call[0])
+    );
+
+    expect(stdoutCalls[2].level).toBe('DEBUG');
+    expect(stdoutCalls[2].meta.graphql).toEqual({
+      operationName: 'createPizza',
+      variables: '{"pizza":{"t […] max payload length reached (12 chars)'
+    });
+
+    logger.destroy();
+  });
+
+  it('logs all `variables` when maxGraphQLVariablesLength = 0', async () => {
```

Comment:
I was [wondering](https://github.com/molindo/molindo-node-logger/pull/21#discussion_r1879828989) if we could preserve this case to turn off variable logging entirely.

So basically 3 cases:
- `maxGraphQlVariablesLength === -1`: Keep all variables
- `maxGraphQlVariablesLength === 0`: No variables
- `maxGraphQlVariablesLength > 0`: Truncate (defaults to 512)

The "no variables" case is currently not possible I think.

What do you think?